composer.json

```diff
@@ -101,7 +101,10 @@
         "psr-4": {
             "yii\\": "framework/",
             "yii\\cs\\": "cs/src/"
-        }
+        },
+        "classmap": [
+            "framework/Yii.php"
```

Comment:
Because the way of framework autoload changed. Also framework not loaded as dependency from composer.